common/lib/api-client/models/person.js

```diff
@@ -1,3 +1,5 @@
+const { _transform, personTransformer } = require('../transformers')
```

Comment:
If we're using `_transform` externally, then it looks weird when its using the "private variable" convention for naming.

Reply:
👍

Reply:
Yeah, I wasn't convinced either and went round a few options for the name...I can't help keep thinking of Optimus Prime each time I was writing them too...

Open to naming suggestions, or maybe just removing the underscore prefix might be the better option for now?